go-sdk/pkg/sdk/keptn.go

```diff
@@ -163,7 +163,11 @@ func (k *Keptn) SendFinishedEvent(event KeptnEvent, result interface{}) error {
 	if err != nil {
 		return err
 	}
-	return k.send(k.createFinishedEventForTriggeredEvent(inputCE, result))
+	if strings.HasSuffix(*event.Type, ".triggered") {
```

Comment:
there is a function in go-utils for checking that: https://github.com/keptn/go-utils/blob/c61381f340919feafa883d216481995ef5734553/pkg/lib/v0_2_0/events.go#L267